src/sbml_jl_model.jl

```diff
@@ -1,43 +1,115 @@
-using SBMLToolkit
-using Catalyst, SBML
-using Test
-
-cd(@__DIR__)
-sbmlfile = joinpath("data", "reactionsystem_01.xml")
-const IV = Catalyst.DEFAULT_IV
-@parameters k1, c1
-@species s1(IV), s2(IV), s1s2(IV)
-
-COMP1 = SBML.Compartment("c1", true, 3, 2.0, "nl", nothing, nothing, nothing, nothing,
-    SBML.CVTerm[])
-SPECIES1 = SBML.Species(name = "s1", compartment = "c1", initial_amount = 1.0,
-    substance_units = "substance", only_substance_units = true,
-    boundary_condition = false, constant = false)  # Todo: Maybe not support units in initial_concentration?
-SPECIES2 = SBML.Species(name = "s2", compartment = "c1", initial_amount = 1.0,
-    substance_units = "substance/nl", only_substance_units = false)
-KINETICMATH1 = SBML.MathIdent("k1")
-KINETICMATH2 = SBML.MathApply("*", SBML.Math[SBML.MathIdent("k1"), SBML.MathIdent("s2")])
-KINETICMATH3 = SBML.MathApply("-",
-    SBML.Math[SBML.MathApply("*",
-            SBML.Math[SBML.MathIdent("k1"),
-                SBML.MathIdent("s1")]),
-        KINETICMATH1])
-REACTION1 = SBML.Reaction(products = [
-        SBML.SpeciesReference(species = "s1", stoichiometry = 1.0),
+using SBML
+
+# Define `Compartment`s
+dummy_compartment = SBML.Compartment(
+    name = "planet_earth",
+    constant = true,
+    spatial_dimensions = 3,
+    size = 1.0,
+)
+
+# Define `Species`
+susceptible = SBML.Species(
+    name = "S",
+    compartment = "planet_earth",
+    initial_amount = 999.0,
+    substance_units = "substance",  # Ignore this for now
+    only_substance_units = true,  # Ignore this for now
+    boundary_condition = false,  # Ignore this for now
+    constant = false,  # Ignore this for now
+)
+# Todo: infected and recovered
+infected = SMBL.Species(
```

Comment:
It is SBML

Reply:
I made those changes 
That's not the problem